src/foam/u2/crunch/PermissionsStringArrayView.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.crunch',
+  name: 'PermissionsStringArrayView',
+  extends: 'foam.u2.View',
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  imports: [
+    'permissionDAO'
+  ],
+
+  requires: [
+    'foam.nanos.auth.Permission',
+    'foam.u2.crunch.PermissionSelection',
+    'foam.u2.TextField'
+  ],
+
+  properties: [
+    {
+      name: 'search',
+      class: 'String',
+      view: {
+        class: 'foam.u2.TextField',
+        type: 'search',
+        placeholder: 'capability search',
+        onKey: true
+      },
+      readVisibility: foam.u2.DisplayMode.RW,//would like it to be accessible even in DAOSummaryView to serch for permissions, but is there any legit way to do that?
```

Comment:
do we need and do we want search to be accessible even for DAOSummaryView?

Reply:
What do you mean? Maybe review + answer questions over zoom today.